client/src/backend/containers/permission/Edit.js

```diff
@@ -64,7 +64,7 @@ export class PermissionEdit extends PureComponent {
   };
 
   removeAllPermissions = () => {
-    event.preventDefault();
+    window.event.preventDefault();
```

Comment:
This line should just be removed, or even should be added as an argument on removeAllPermissions handler if it's responding to an event.

Reply:
So it's safe to just delete this line? I'm not sure what type of event you would like to have trigger a preventDefault.